app/frontend/src/administrative/settings/settings.component.html

```diff
@@ -0,0 +1,184 @@
+<div id="sitnet-header" class="header">
+    <div class="col-md-12 header-wrapper">
+        <span class="header-text">{{ 'sitnet_settings' | translate }}</span>
+    </div>
+</div>
+
+<div id="dashboard">
+    <div class="top-row" *ngIf="config">
+        <div class="col-md-12 form-group">
+            <label for="editor">{{ 'sitnet_user_agreement' | translate }}</label>
+            <textarea id="editor" name="editor" ck-editor rows="10" cols="80" [(ngModel)]="config.eula"> </textarea>
```

Comment:
Tässä vois käyttää utility-moduulin ck-editor-komponenttia tyyliin näin:

`<ckeditor #ck="ngModel" ...></ckeditor>`